packages/eui/src/components/form/form_row/form_row.stories.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiFieldText } from '../field_text';
+import { EuiForm } from '../form';
+
+import { EuiFormRow, EuiFormRowProps } from './form_row';
+
+const meta: Meta<EuiFormRowProps> = {
+  title: 'Forms/EuiForm/Subcomponents/EuiFormRow',
```

Comment:
```suggestion
  title: 'Forms/EuiFormRow',
```

This is super confusing but `EuiFormRow` and other form control components are actually **not** subcomponents of `EuiForm` (as in, they can be and are frequently used standalone outside of EuiForm, which is how I personally mentally differentiate between 'subcomponents' and sibling components - i.e., how much of a required parent/child relationship they have).

This would be my personal suggestion for hierarchy:

- EuiForm
- EuiFormRow
  - Subcomponents (Typically only used within EuiFormRow)
    - EuiFormErrorText
    - EuiFormHelpText
    - EuiFormLabel
- EuiDescribedFormGroup
- EuiFormFieldset
- EuiFormControlLayout
- EuiFormControlLayoutDelimited

WDYT? Does that make sense to you given our current form docs, or am I way off?

Reply:
Ah yeah I think that makes sense! I wasn't sure about EuiFormRow being a subcomponent in the first place considered none of the others seemed to be 😅  I'm fine with updating to your suggestion 👍

Reply:
The hierarchy was updated in this [commit](https://github.com/elastic/eui/pull/7790/commits/d7aa4543b3eabcd39352e3028bd3b2628c1c1f7c).